src/otx/v2/api/core/dataset.py

```diff
@@ -145,8 +149,8 @@ def __init__(
         """
 
     @abstractmethod
-    def get_otx_dataset(self) -> DatasetEntity:
-        """Get DatasetEntity."""
+    def get_otx_dataset(self) -> dict[Subset, DatumDataset]:
+        """Get Dict[Subset, DatumDataset]."""
```

Comment:
Is this docstring intentional?

Reply:
I just followed the original docstring format. I think we have to update all docstrings when the code is ready finally.